docs/development/ideConfig.md

```diff
@@ -45,6 +45,7 @@
    Uncheck the `use default` for the startup script and select the `startupwithenv.sh` script from the project root.
    Next add a new env var called `APP_ENV` and set the value to `devlocal`.
    ![zac-intellij-runtime-wildfly-4.png](./attachments/images/zac-intellij-runtime-wildfly-4.png)
+6. Make sure you configured 1Password startup script for both `Run` and `Debug`
```

Comment:
Clear! On the other hand: why would you ever use `Run`? If you just want to run ZAC you should just run it in Docker? The only purpose of running ZAC in IntelliJ is to be able to debug I would think? But this is fine. :-)

Reply:
It's convenient for checks, tests ...
But yes. Debug is the main purpose indeed

However from the docu it seems we target Run only, so decided to make this explicit